TODO.md

```diff
@@ -0,0 +1,13 @@
+## TODO
+- [ ] Implement conways logic
+    - [ ] Implement life rules 
+    - [ ] Use life rules to change state of cells
+- [ ] Game frontend
+    - [ ] Implement main event loop logic
+- [ ] Add rust docs
+- [ ] Make clippy happy
+- [x] Add README
+- [x] Add Makefile 
+
+
+
```

Comment:
Very minor nitpick, but it's better if files don't have extra newlines at the end.

1 is fine, and often required. More than that is a bit overkill